Implement dev tools support for `wxWebViewChromium`

vadz opened the pull request:
Add context menu items for showing/hiding CEF developer tools and inspecting the current element to the context menu if access is enabled.

hydrozoan committed:
Implement wxWebViewChromium::EnableAccessToDevTools()

Add context menu items for showing/hiding CEF developer tools and
inspecting the current element to the context menu if access is enabled.

Co-authored-by: Vadim Zeitlin <vadim@wxwidgets.org>

PBfordev commented on src/common/webview_chromium.cpp:
```diff
@@ -1793,6 +1863,18 @@ CefRefPtr<CefResourceHandler> ClientHandler::GetResourceHandler(
 }
 
+void
+ClientHandler::ShowDevTools(CefRefPtr<CefBrowser> browser,
```
Is there a reason why basic `ShowDevTools()` is not public? I think it may be handy to be able to show the tool programmatically.

The `wxWebViewEdge` and `wxWebViewWebKit` also support this, see https://github.com/PBfordev/wxecharts/blob/c3f50dc464308b623272f345b06071a1212f2f4a/mainframe.cpp#L308. I can't use macOS so I don't know about that one.

vadz replied:
Looks like a good idea, thanks, I think I can add it. For now let's keep it simple and just allow opening it without specifying the position of the element to inspect, we can always extend it later if really necessary.

vadz committed:
Add wxWebView::ShowDevTools()

Implement it for Edge, WebKit2GTK and Chrome backends.

It doesn't seem possible to implement this for WebKit macOS backend and
IE backend doesn't support dev tools at all.

Also note that Edge backend doesn't seem to support closing the dev
tools window, so don't provide this functionality for now, even if it
could be implemented for WebKit2 and Chrome backends.
